src/main/java/at/hannibal2/skyhanni/mixins/hooks/MouseSensitivityHook.kt

```diff
@@ -0,0 +1,72 @@
+package at.hannibal2.skyhanni.mixins.hooks
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.api.event.HandleEvent
+import at.hannibal2.skyhanni.events.DebugDataCollectEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+
+@SkyHanniModule
+object MouseSensitivityHook {
```

Comment:
the hook should be kept clean. if you dont wanna merge all those members and functions into the garden related sensitivity reducer, then please create a new class "mouse sensitivity" or similar, and include the name garden somehow in the sensitivy reducer file name